src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -859,6 +881,13 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
         cmd_helm_upgrade.extend(["--kubeconfig", kube_config])
     if kube_context:
         cmd_helm_upgrade.extend(["--kube-context", kube_context])
+    if container_registry_repository:
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.image.repository={}".format(container_registry_repository)])
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.image.username={}".format(container_registry_username)])
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.image.password={}".format(container_registry_password)])
+    else:
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.image.repository={}".format("mcr.microsoft.com")])
```

Comment:
So, does this mean if an existing pcr customer runs this cmd to update proxy params(let's say) , he will be switched back to mcr? Dosen't look like the right experience. Is the expectations that the PCR customer always passes pcr params even if they want to update something else(eg: proxy params)?

Reply:
added --default-registry parameter to switch back to mcr